deps/build.jl

```diff
@@ -199,6 +199,36 @@ else # source build
    cd(wdir)
 end
 
+# add shell scripts that startup another julia for some lib4ti2 programs
+# singular and libsingular are supposed to at least look in
+#  $prefixpath/lib/singular/MOD
+
+mkpath("$prefixpath/lib/singular/MOD")
```

Comment:
Shouldn't the directory already exist, if Singular is adding it to its PATH?

Reply:
I think the point here is that (lib)singular look in this path *if it exists*, which it doesn't by default.